hack/cluster-deploy.sh

```diff
@@ -87,6 +87,26 @@ if [[ "$KUBEVIRT_DEPLOY_CDI" != "false" ]] && [[ ${ARCHITECTURE} != *aarch64 ]];
     _ensure_cdi_deployment
 fi
 
+if [[ "${KUBEVIRT_DEPLOY_NFS_CSI}" == "true" ]]; then
+    # Hack to change the nfs-csi storage class outside of kubevirtci/gocli
+    _kubectl delete sc nfs-csi
+    _kubectl create -f - <<EOF
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: nfs-csi
+provisioner: nfs.csi.k8s.io
+parameters:
+  server: nfs
```

Comment:
I guess something on the node (kvci VM) knows how to translate "nfs"?

Reply:
Yeah... I did the same as other things that use the server, like https://github.com/kubevirt/kubevirt/blob/main/automation/test.sh#L388

Reply:
I see so likely this https://github.com/kubevirt/kubevirtci/blob/47782438418df775cd5db27f0a7b5435c497b404/cluster-provision/gocli/containers/dnsmasq.go#L66